src/trex_timesync.cpp

```diff
@@ -119,3 +117,7 @@ const char *CTimesyncEngine::descTimesyncState(int port) {
     }
     return "NONE";
 }
+
+timespec CTimesyncEngine::timestampToTimespec(uint64_t timestamp) {
+    return {(uint32_t)(timestamp / (1000 * 1000 * 1000)), (uint32_t)(timestamp % (1000 * 1000 * 1000))};
+};
```

Comment:
you can add nl

Reply:
auto-formatting...

Reply:
github is angry because no newline at the eno of file

Reply:
I have fed him.  Should be calmer now.